app/controllers/admin/auctions_controller.rb

```diff
@@ -3,7 +3,7 @@ class AuctionsController < ApplicationController
     before_filter :require_admin
 
     def index
-      @auctions = Auction.all.map { |auction| Presenter::AdminAuction.new(auction) }
+      @auctions = Auction.all.map { |auction| AdminAuctionPresenter.new(auction) }
```

Comment:
So, we want to switch to a different naming style? I don't have a preference (and this might avoid some issues with Rails' autoload), but I want to see what we all think